templates/CRM/ACL/Header.tpl

```diff
@@ -13,40 +13,3 @@
 <div class="help">
   <p>{ts 1=$docLink}ACLs allow you to control access to CiviCRM data. An ACL consists of an <strong>Operation</strong> (e.g. 'View' or 'Edit'), a <strong>set of data</strong> that the operation can be performed on (e.g. a group of contacts), and a <strong>Role</strong> that has permission to do this operation. Refer to the %1 for more info.{/ts}</p>
 </div>
-
-{php}
-  $currentStep = $this->getTemplateVars('step');
-  $wizard = [
-    'style' => ['barClass' => '', 'showTitle' => FALSE],
```

Comment:
I signed up for a bar class, but my style still came out empty. I probably studied under the wrong teacher.

![](https://hips.hearstapps.com/hmg-prod/images/lead-1547849001.jpg)

Reply:
@totten you got shorter ^^

Reply:
@eileenmcnaughton For that hair, it's an even trade.

Reply:
lol